app/etc/config.php

```diff
@@ -77,7 +77,7 @@
     'Magento_Robots' => 1,
     'Magento_Rss' => 1,
     'Magento_CatalogRuleConfigurable' => 1,
-    'Magento_Authorizenet' => 1,
```

Comment:
I dont think I have removed that modules manually 
I removed my eav_attribute and setup_module and then re-se:up